tools/autodeployment/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [ $1 = 'production']
+then
+  DOCKER_FILE=docker-compose-production.yml
+else
+  DOCKER_FILE=docker-compose-staging.yml
+fi
```

Comment:
Let's do 2 things here:

1. add a specific check for `$1 = 'staging'` vs using else to assume.
1. if it's neither of these two, the else should print an error message and exit.

Reply:
bash newbie here, let me know if my pushed change is what you were expecting

Reply:
added a check, also took @manekenpix suggestion to set the ENV_FILE var here instead of taking a second argument